.circleci/config.yml

```
@@ -168,6 +168,20 @@ jobs:
           command: make citest fork=eip6110
       - store_test_results:
           path: tests/core/pyspec/test-reports
+  test-eip7045:
+    docker:
+      - image: circleci/python:3.8
```

```suggestion
      - image: circleci/python:3.9
```

Plus, it would be better to run `git merge origin dev` which will fix eip6110 CI job.